code/places-monitor-android/src/phone/java/com/adobe/marketing/mobile/PlacesMonitorLocationPermission.java

```diff
@@ -40,7 +41,12 @@
 	/**
 	 * Permission for Places Monitor to access location in foreground and background.
 	 */
-	ALWAYS_ALLOW("alwaysAllow");
+	ALWAYS_ALLOW("alwaysAllow"),
```

Comment:
are the string values for this enum important to the platform, or can we use the same strings we are using in ios? 

    std::string AUTH_UNKNOWN = "unknown";
    std::string AUTH_RESTRICTED = "restricted";
    std::string AUTH_DENIED = "denied";
    std::string AUTH_ALWAYS = "always";
    std::string AUTH_WHEN_IN_USE = "wheninuse";

Reply:
This is the enum for PlacesRequestLocationPermission
https://github.com/adobe/places-monitor-ios/blob/master/ACPPlacesMonitor/ACPPlacesMonitor.h#L49

Reply:
This is the enum for request location permission..

similar to one we have in iOS below
https://github.com/adobe/places-monitor-ios/blob/master/ACPPlacesMonitor/ACPPlacesMonitor.h#L49

enum name is a little bit different